servicebox-app/src/main/java/io/halkyon/ClaimCustomResource.java

```diff
@@ -37,4 +38,12 @@ private TemplateInstance showList(List<Claim> claims) {
     public Claim get(String name) {
         return Claim.findByName(name);
     }
+
+    @POST
+    @Transactional
+    public Response add(@Form Claim claim) {
+        claim.persist();
+        return Response.status(Response.Status.FOUND).header("Location", "/claims").build();
+        //return Response.ok(Templates.item(claim)).header("Location", "/claims").build();
```

Comment:
hx-swap="beforeend" would require you to return the HTML representation of that new claim. The commented code seems ok, I would use that instead of returning only the status.